examples/oneapi/cpp/source/graph/graph_service_functions.cpp

```diff
@@ -28,19 +28,20 @@ using namespace oneapi::dal::preview;
 int main(int argc, char **argv) {
     const std::string filename = get_data_path("graph.csv");
 
-    graph_csv_data_source ds(filename);
-    load_graph::descriptor<> d;
-    auto my_graph = load_graph::load(d, ds);
+    const graph_csv_data_source ds(filename);
+    using my_graph_type = undirected_adjacency_array_graph<>;
+    const load_graph::descriptor<edge_list<int32_t>, my_graph_type> desc;
+    const auto my_graph = load_graph::load(desc, ds);
     std::cout << "Number of vertices: " << get_vertex_count(my_graph) << std::endl;
     std::cout << "Number of edges: " << get_edge_count(my_graph) << std::endl;
 
-    auto node_id = 0;
-    std::cout << "Degree of " << node_id << ": " << get_vertex_degree(my_graph, node_id)
+    vertex_type<my_graph_type> vertex_id = 0;
```

Comment:
you make `using namespace oneapi::dal` and `using namespace oneapi::dal::preview`. Do not we want to mark in exaples which functionality is preview and which is not? Suggest not to use `oneapi::dal::preview` namespace and explicitly mark every preview feature `preview::vertex_type<my_graph_type>`